string tphsmm_name
string path_to_data
int8 num_demos #How many of the demos from path_to_data do you want to use
int8 num_frames #Number of important frames of reference in the demonstrations
